website/docs/08_glossary/01_devops.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: DevOps
```

Comment:
all glossary pieces should also have a description (image might be nice, but can be added as step two)

Reply:
I've added a description but will leave the image to another step since I'm not sure the best way to visualize it.